magento/magento-cloud-patches
# MAGECLOUD-4980: Pagebuilder patches fail CE functional tests

### Description

Pagebuilder patches can not apply on CE as Pagebuilder  module is not present.
### Fixed Issues (if relevant)

1. Moved constraint to be on magento2-ee-base instead of magento2-base

### Manual testing scenarios

1. Confirm any version of patch still applies against EE.  Check automated CE tests following merge.

### Contribution checklist
